src/components/DevicesModaleRevokeView.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { translate } from 'cozy-ui/react/helpers/i18n'
+import Modal from 'cozy-ui/react/Modal'
+
+const devicesModaleRevokeView = ({ t, device, revokeDevice, cancelAction }) => {
+  return (<Modal
+    title={t('revokeDevice.title')}
+    description={t('revokeDevice.description', {name: device.client_name})}
+    subText={t('revokeDevice.subText')}
```

Comment:
The Modal component doesn't support a `subText` property, so it's not showing up as far as I can tell.
The correct approach here is to create the DOM structure you want as a description before hand, and pass it all in the `description` prop. [Here's an example](https://github.com/cozy/cozy-files-v3/blob/master/src/containers/DeleteConfirmation.jsx#L13) of how to do that.